Source/WebCore/css/CSSProperties.json

```diff
@@ -1557,11 +1557,13 @@
             }
         },
         "animation-range": {
```

Comment:
This should be set a longhand of `animation`, where we reset `animation-range` to the default value

Reply:
This should probably be a separate patch. I think animation-timeline also needs a similar fix.

Reply:
The `animation-timeline` property is not part of the `animation` shorthand (see [bug 279732](https://bugs.webkit.org/show_bug.cgi?id=279732)) and I don't believe `animation-range` is either.